mteb/models/openai_models.py

```diff
@@ -19,8 +19,8 @@ def __init__(
         self,
         model_name: str,
         max_tokens: int,
```

Comment:
```suggestion
        model_name: str,
        *,
        max_tokens: int,
```

I would do this to ensure that every argument is a keyword argument.